data/data_utils.py

```diff
@@ -0,0 +1,65 @@
+import pickle
```

Comment:
I would suggest to have a few lines written at the top of every new file, describing what to expect here, e.g.,
"""
This module is responsible for reading json-line data and its tokenization.
"""

This could be done in order to make the structure more understandable and easier for memory refresh once we forget what was developed in a specific part of the repo :D